CHANGELOG.md

```diff
@@ -6,7 +6,7 @@ All notable changes to this project will be documented in this file. If a contri
 
 ## Unreleased
 
-- n/a
+- Deprecate experimental preprocess mode see [#1619](https://github.com/styled-components/styled-components/issues/1619) - @Samatar26
```

Comment:
Let's match the other entries, i.e.

```
- Add XYZ, by [@Samatar26](https://github.com/Samatar26) (see [#1619](https://github.com/styled-components/styled-components/issues/1619))
```

meaning, let's add the ", by X" with a link instead and put the "see" link into parens :wink: